app/[contentType]/page.tsx

```diff
@@ -0,0 +1,65 @@
+import { Metadata } from 'next';
+import { notFound } from 'next/navigation';
+import { Container, ContentList } from '../../components';
+import { site } from '../../config/index.json';
+import {
+  getContentList,
+  getContentTypes,
+  IContentType,
+} from '../../utils/content';
+import { contentTypesMap } from '../../utils/content-types';
+import { generateRSS } from '../../utils/rss';
+
+type Params = {
+  contentType: IContentType;
+};
+
+/** generate list page metadata */
+export function generateMetadata({ params }: { params: Params }): Metadata {
+  const contentType = contentTypesMap.get(params.contentType);
```

Comment:
Add type `Params` and  change `CONTENT_TYPES_MAP` to camelCase `contentTypesMap`

Reply:
is the `Params` type used more than once? If not, maybe inline it like you did elsewhere?

Reply:
Yes, `Params` is used two times in the same file. This is the case in 2 files: 
- `app/[contentType]/page.tsx`
- `app/[contentType]/[slug]/page.tsx`